infermedica_api/models/diagnosis.py

```diff
@@ -117,22 +117,25 @@ def __init__(self, sex, age, time=None, interview_id=None, **kwargs):
     def observations(self):
         return self.symptoms
 
-    def __add_evidence(self, collection, _id, state, time, initial):
+    def __add_evidence(self, collection, _id, state, time, initial=None, source=None):
         """Helper function to update evidence list."""
         evidence = {
             "id": _id,
             "choice_id": state
         }
-
+        if initial is not None:
+            warnings.warn(
+                "Parameter initial is deprecated, please use source with following parameter: initial, suggest, "
+                "predefined, red_flags.", category=DeprecationWarning
+            )
         if time:
             evidence['observed_at'] = time
-
-        if initial:
-            evidence['initial'] = True
+        if source:
+            evidence['source'] = source
 
         collection.append(evidence)
 
-    def add_observation(self, _id, state, time=None, initial=None):
+    def add_observation(self, _id, state, time=None, source=None):
```

Comment:
Please restore the initial parameter in other places as well.

Reply:
@akszydelko done